store/datastore/repos_test.go

```diff
@@ -138,6 +139,52 @@ func TestRepos(t *testing.T) {
 			g.Assert(count).Equal(2)
 		})
 
+		g.It("Should Get a Repo List Paginated", func() {
+			_repos := []*model.RepoLite{
+				{FullName: "bradrydzewski/drone"},
```

Comment:
the newer versions of the Go linter will complain when underscores are used in variable names, so we've tried to start removing these from our code. So perhaps `repoLiteList` instead?